### PR TITLE
task/view_images/implement_adapter

### PR DESCRIPTION
**Pull request by teamagam1**
Created adapters

### CODE REVIEW

File: DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureAdapter.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import java.util.Collections;
+import java.util.List;
+
+class PictureAdapter extends RecyclerView.Adapter<PictureAdapter.PictureViewHolder> {
+    private LayoutInflater mInflater;
+    private List<PictureInfo> mPictureInfoList = Collections.emptyList();
```
Comment (BarAgam) [inline DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureAdapter.java]: Seems like this initialization is redundant, the field is set at the constructor.

File: DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureAdapter.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+    PictureAdapter(Context context, List<PictureInfo> pictureInfoList) {
+        mInflater = LayoutInflater.from(context);
+        mPictureInfoList = pictureInfoList;
+    }
+
+    @Override
+    public PictureViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = mInflater.inflate(R.layout.layout_picture, parent, false);
```
Comment (BarAgam) [inline DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureAdapter.java:25]: I don't know if there's any difference, but you can get an inflater using:
LayoutInflater.from(parent.getContext())
Just FYI

File: DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureInfo.java [hunk 3]
```diff
@@ -0,0 +1,6 @@
+package com.teamagam.dailyselfie;
+
+class PictureInfo {
+    String mPath;
```
Comment (BarAgam) [inline DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureInfo.java]: We actually don't have a coding convention for public fields naming, but the m prefix is usually used to denote a private field member.

Comment (BarAgam) [reply DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureInfo.java]: You could go with just "path" and "filename", but it's up to you.

**Review by BarAgam — APPROVED** [review]
Nice and clean 👍

File: DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureInfo.java (hunk 3 above)
Comment (teamagam5) [reply DailySelfie/app/src/main/java/com/teamagam/dailyselfie/PictureInfo.java]: Totally right, my mistake

**Pull request merged**